rpc/src/v1/impls/eth.rs

```diff
@@ -672,18 +672,22 @@ impl<C, M> EthFilter for EthFilterClient<C, M> where
 							to_value(&hashes)
 						},
 						PollFilter::PendingTransaction(ref mut previous_hashes) => {
+							// get hashes of pending transactions
 							let current_hashes = take_weak!(self.miner).pending_transactions_hashes();
-							// calculate diff
 							let previous_hashes_set = previous_hashes.into_iter().map(|h| h.clone()).collect::<HashSet<H256>>();
```

Comment:
doesn't work here, cause  `error: type mismatch resolving `<core::slice::IterMut<'_, util::hash::H256> as core::iter::Iterator>::Item == &_`: values differ in mutability`

same with `map(Clone::clone)`